lib/uo_cli.rb

```diff
@@ -198,13 +198,13 @@ def opt_visualize
         @subopts = Optimist.options do
           banner "\nURBANopt #{cmd}:\n \n"
 
-          opt :scenarios, "\nVisualize results for all scenarios\n" \
+          opt :feature, "\nVisualize results for all scenarios for a feature file\n" \
             "Provide the FeatureFile whose scenario results you want to visualize\n" \
```

Comment:
"Provide the FeatureFile to visualize each associated scenario."

also no default here since we don't know the project dir